code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -50,6 +50,65 @@
 		var/chosen_sound = pick(migo_sounds)
 		playsound(src, chosen_sound, 100, TRUE)
 
+/mob/living/simple_animal/hostile/netherworld/imlagre
+	name = "imlagre"
+	desc = "An odd looking blue fellow that always comes in <span class='danger'>3</span>."
+	icon_state = "imlagre4"
+	icon_living = "imlagre4"
+	icon_dead = "imlagredead"
+	gold_core_spawnable = NO_SPAWN
+	health = 50
+	maxHealth = 50
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	attacktext = "slashes"
+	deathmessage = "unwinds in a a paroxysm of laughter."
+	var/list/linked_imlagre = list()
+	var/laughmod = 1 //need this on the mob so it carries over to life, and i like letting admins var as much as they please
+	var/list/laughs = list('sound/voice/human/manlaugh1.ogg', 'sound/voice/human/manlaugh2.ogg')
+
+/mob/living/simple_animal/hostile/netherworld/imlagre/Initialize(mapload, initial = TRUE)
+	. = ..()
+	laughmod = rand(0.5,1.5) //they laugh the same every time but it's a random pitch and there are three of them with random pitches so it works out
+	if(initial) //use this var to check if it's the first to spawn
+		var/turf/T = get_turf(src) //cache for speed
+		var/mob/living/simple_animal/hostile/netherworld/imlagre/one = new(T, FALSE) //maintainers no N- BANG BANG BANG AAAAAAH MY GUTS AAAAAAH BANG SPLAT
+		var/mob/living/simple_animal/hostile/netherworld/imlagre/two = new(T, FALSE) //ok but really i need the two vars so i can link everything there's a reason i'm not doing for(var/i in 1 to 2)
+		linked_imlagre.Add(one)
+		linked_imlagre.Add(two)
+		one.linked_imlagre.Add(src)
+		one.linked_imlagre.Add(two)
+		two.linked_imlagre.Add(src)
+		two.linked_imlagre.Add(one)
```

Comment:
I showed him how to change this to also allow for a variety of linked imlagre amounts